ui/components/src/StatChart/StatChart.tsx

```diff
@@ -24,23 +23,14 @@ import { formatValue, UnitOptions } from '../model/units';
 import { EChart } from '../EChart';
 import { GraphSeries } from '../model/graph';
 
-use([
-  EChartsGaugeChart,
-  EChartsLineChart,
-  GridComponent,
-  DatasetComponent,
-  TitleComponent,
-  TooltipComponent,
-  CanvasRenderer,
-]);
+use([EChartsLineChart, GridComponent, DatasetComponent, TitleComponent, TooltipComponent, CanvasRenderer]);
 
 const MIN_VALUE_SIZE = 12;
 const MAX_VALUE_SIZE = 36;
 
 export interface StatChartData {
   calculatedValue?: number;
   seriesData?: GraphSeries;
-  name?: string;
```

Comment:
@eunicorn I cleaned up a few types that weren't used anymore in this chart while I was making adjustments to support `null`, sorry for any merge conflicts with your thresholds branch!